### PR TITLE
Add repository and mappings for embeddable vo

### PR DESCRIPTION
**Pull request by fabiocarneiro**

### CODE REVIEW

File: config/module.config.php [hunk 1]
```diff
@@ -4,6 +4,41 @@
...
-return [
+use Doctrine\Common\Persistence\Mapping\Driver\PHPDriver;
+use ZFBrasil\RepositoryManager\DBAL\Types\RepositoryPath;
+use ZFBrasil\RepositoryManager\DBAL\Types\RepositoryType;
+use ZFBrasil\RepositoryManager\Model;
 
+return [
+    'doctrine' => [
+        'driver' => [
+            'repositorymanager_driver' => [
```
Comment (danizord) [inline config/module.config.php]: Use module namespace here

File: config/module.config.php [hunk 2]
```diff
@@ -4,6 +4,41 @@
...
+                'class' => PHPDriver::class,
+                'paths' => [
+                    __DIR__ . '/../src/Model/Mapping'
+                ]
+            ],
+            'orm_default' => [
+                'drivers' => [
+                    Model::class => 'repositorymanager_driver'
```
Comment (danizord) [inline config/module.config.php]: Same here

File: config/module.config.php [hunk 3]
```diff
@@ -4,6 +4,41 @@
...
+                ],
+            ],
+        ],
+        'connection' => [
+            'orm_default' => [
+                'doctrine_type_mappings' => [
+                    'repository_path' => 'repository_path',
```
Comment (danizord) [inline config/module.config.php]: FQCN

File: config/module.config.php (hunk 1 above)
Comment (fabiocarneiro) [inline config/module.config.php]: it was done in #3

File: config/module.config.php (hunk 2 above)
Comment (fabiocarneiro) [inline config/module.config.php]: it was already done in #3

File: config/module.config.php (hunk 3 above)
Comment (fabiocarneiro) [inline config/module.config.php]: it was already done in #3

File: config/module.config.php (hunk 1 above)
Comment (danizord) [inline config/module.config.php]: But we'll merge this PR first, or not?

File: config/module.config.php (hunk 3 above)
Comment (danizord) [inline config/module.config.php]: Ah nvm

File: src/DBAL/Types/RepositoryPath.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * @param mixed            $value
+     * @param AbstractPlatform $platform
+     *
+     * @return mixed|Path
```
Comment (danizord) [inline src/DBAL/Types/RepositoryPath.php]: null|Path

File: src/DBAL/Types/RepositoryType.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * @param mixed            $value
+     * @param AbstractPlatform $platform
+     *
+     * @return mixed|Type
```
Comment (danizord) [inline src/DBAL/Types/RepositoryType.php]: null|Type

File: test/RepositoryManagerTest/DBAL/Types/RepositoryPathTest.php [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+ */
+class RepositoryPathTest extends TestCase
+{
+    /**
+     * Register the global dbal type
+     */
+    public function setUp()
+    {
+        if (false === Type::hasType('repository_path')) {
+            Type::addType('repository_path', RepositoryPath::class);
```
Comment (danizord) [inline test/RepositoryManagerTest/DBAL/Types/RepositoryPathTest.php]: You de we need this?

Comment (fabiocarneiro) [inline test/RepositoryManagerTest/DBAL/Types/RepositoryPathTest.php]: You can't test the code in tests below if you dont have a configured type with that name.

**Pull request merged**